databases/week2/DB_Week2.sql

```diff
@@ -0,0 +1,136 @@
+-- Part 1: Working with tasks
+INSERT INTO user (name, email) 
+VALUES ('Rostyslav', 'rostik94@gmail.com');
+
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id) 
+VALUES ('Do the homework', 'Week2 homework', '2022-07-04', '2022-07-04', '2022-07-10', 1, 13);
+
+UPDATE task set title = 'Database Week2 Homework'
+Where id = 36;
+
+UPDATE task set due_date = '2022-07-04'
+WHERE id = 36;
+
+UPDATE task set status_id = 2
+WHERE id = 36;
+
+UPDATE task set status_id = 3
+WHERE id = 36;
+
+DELETE FROM task WHERE id = 36;
+
+-- Part 2: School database
+CREATE DATABASE school
+    DEFAULT CHARACTER SET = 'utf8mb4';
+
+USE school;
+
+CREATE TABLE class (
+    id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+    class_name VARCHAR(255) NOT NULL,
+    begins DATETIME NOT NULL,
+    ends DATETIME NOT NULL
+);
+
+CREATE TABLE student (
+    id INT UNSIGNED AUTO_INCREMENT,
+    student_name VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL unique,
+    phone VARCHAR(255) unique,
+    class_id INT UNSIGNED NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (class_id) REFERENCES class(id)
+);
+
+CREATE INDEX idx_name
+ON student (student_name);
+
+
+ALTER TABLE student
+ADD status ENUM ('not-started', 'ongoing', 'finished') NOT NULL;
+
+-- Part 3: More queries
+SELECT task.title AS 'Task Name', user.name AS 'User Name' FROM user_task
+JOIN task ON user_task.task_id = task.id
+JOIN user ON user_task.user_id = user.id
+WHERE user.email LIKE '%@spotify.com%'; 
+
+SELECT task.title AS 'Task Name', user.name AS 'User Name', status.name AS 'Task Status' FROM user_task
+JOIN task ON user_task.task_id = task.id
+JOIN user ON user_task.user_id = user.id
+JOIN status ON task.status_id = status.id
+WHERE user.name = 'Donald Duck' 
+AND task.status_id = 1; 
+
+SELECT task.title AS 'Task Name', user.name AS 'User Name' FROM user_task
+JOIN task ON user_task.task_id = task.id
+JOIN user ON user_task.user_id = user.id
+WHERE user.name = 'Maryrose Meadows' 
+AND month(task.created) = 9; 
+
+SELECT COUNT(task.id) AS 'Tasks amount', MONTH(task.created) AS 'Task Created Month' FROM task
+GROUP BY MONTH(task.created);
+
+-- Part 4: Creating a database
+
+CREATE DATABASE bilka_db
```

Comment:
First things first, you didn't include any many-to-many relationships, which was a requisite of the homework, so you need to do that.

Also, some questions, and i'm **not** saying something is wrong: does one department only have one store? Does one employee only work in one department? 

The way i see things, is that different stores have the same electronics department for instance, and one employee might eventually work in multiple departments? I could of course be wrong, especially in the latter one.